projects/Apps/editions-src/src/helpers/__tests__/files.spec.ts

```diff
@@ -28,19 +28,19 @@ const apolloClientMock: ApolloClient<object> = {
 } as any
 
 describe('helpers/files', () => {
-    describe('matchSummmaryToKey', () => {
-        it('should return a matched IssueSummary if the key matches', () => {
-            const key = 'daily-edition/2019-09-18'
-            const isValidIssueSummary = matchSummmaryToKey(issueSummaries, key)
-            expect(isValidIssueSummary).toEqual(issueSummaries[0])
-        })
+    // describe('matchSummmaryToKey', () => {
+    //     it('should return a matched IssueSummary if the key matches', () => {
+    //         const key = 'daily-edition/2019-09-18'
+    //         const isValidIssueSummary = matchSummmaryToKey(issueSummaries, key)
+    //         expect(isValidIssueSummary).toEqual(issueSummaries[0])
+    //     })
 
-        it('should return null if the key doesnt match', () => {
-            const key = 'daily-edition/2019-09-20'
-            const isValidIssueSummary = matchSummmaryToKey(issueSummaries, key)
-            expect(isValidIssueSummary).toEqual(null)
-        })
-    })
+    //     it('should return null if the key doesnt match', () => {
+    //         const key = 'daily-edition/2019-09-20'
+    //         const isValidIssueSummary = matchSummmaryToKey(issueSummaries, key)
+    //         expect(isValidIssueSummary).toEqual(null)
+    //     })
+    // })
```

Comment:
shall we remove these tests completely or ad a comment explaining why they're not in use?

Reply:
I've added a comment